packages/cicero-tools/lib/codegen/fromcto/corda/cordaplugin.js

```diff
@@ -57,7 +57,8 @@ class CordaPlugin extends AbstractPlugin {
         if(clazz.getFullyQualifiedName() === 'org.accordproject.money.MonetaryAmount') {
             parameters.fileWriter.writeLine(1, `
    public Amount<Currency> getCurrency() {
-      return Currencies.DOLLARS(doubleValue);
```

Comment:
If `Currencies` is not used anymore, the corresponding `import net.corda.finance.Currencies;` should be removed as well.